frontend/packages/dev-console/src/components/topology/components/groups/Application.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@patternfly/patternfly/sass-utilities/colors';
+@import '../../../../../../../node_modules/@patternfly/patternfly/sass-utilities/colors';
```

Comment:
Yikes... what happened here? Is this really required? This looks horrible lol.

Reply:
not sure how that happened, works fine the original way. Sorry I missed that.